passwd.master

```diff
@@ -42,4 +42,4 @@ g-s-helper:*:122:129:GNOME Software helper system user,,,:/var/lib/g-s-helper/:/
 ndn-user:*:123:130::/var/lib/ndn:/bin/false
 systemd-network:*:124:131:systemd Network Management,,,:/run/systemd/netif:/bin/false
 systemd-resolve:*:125:132:systemd Resolver,,,:/run/systemd/resolve:/bin/false
-companion-app-helper:*:126:133:Companion Application helper system user,,,:/var/lib/companion-app/:/usr/sbin/nologin
+companion-app-helper:*:126:134:Companion Application helper system user,,,:/var/lib/companion-app/:/bin/false
```

Comment:
It strikes me that this is not the best way to achieve what you need...
The `companion-app-helper` user should have the `companion-app-helper` group as the main group, and also be added to the `avahi-service-writers`. So you should keep this here as it is, and add it to the other group through some other mean (e.g. the `postinst` file)

Reply:
Would it make sense to add a `usermod -a -G avahi-service-writers companion-app-helper` to the eos-companion-app-helper